packages/cli/internal/pkg/cli/logs_core.go

```diff
@@ -132,10 +134,15 @@ func (o *logsSharedOpts) displayEventFromChannel(channel <-chan cwl.StreamEvent)
 			for _, line := range event.Logs {
 				printLn(line)
 			}
+			firstEvent = false
+		} else if firstEvent {
+			firstEvent = false
+			printLn("There are no new logs. Please wait for the first logs to appear...")
```

Comment:
Can this be an info level log? Since it goes to stderr users can capture just the cwl logs from stdout